parser.go

```diff
@@ -2519,7 +2669,7 @@ func (p *Parser) tryParseInterleaveIn() *ast.InterleaveIn {
 }
 
 func (p *Parser) parseAlterTable(pos token.Pos) *ast.AlterTable {
-	p.expectKeywordLike("TABLE")
+	p.nextToken()
```

Comment:
What is this change?

Reply:
Due to this change https://github.com/cloudspannerecosystem/memefish/pull/58/files/9b3a4cb38877bf8fe17cb64fba73d1ae297ab648#diff-83eb8e32639d01cf443d6d8bde24c1c8be78766090d8c5f8586c36250cfedca6R1997-R2002 ,
I changed to check token after `ALTER` token in `parseDDL`.
When next token is `TABLE`, then we call `parseAlterTable()` with `p.Token.Pos` points `TABLE`, when next tokens are `CHANGE STREAM`, then we call `parseAlterChangeStream()`. 

 Is this change acceptable?

Reply:
No. If `p.Token` is set to a specific value when `p.parseAlterTable` is called, then `p.expectKeywordLike` should all the more be used to make it clear.